data/core/units/dunefolk/Wyvern_Rider.cfg

```diff
@@ -14,7 +14,7 @@
     {AMLA_DEFAULT}
     cost=102
     usage=scout
-    description=""
+    #description=""
```

Comment:
I feel a `[unit_type]` that lacks a description should fail validation.

Reply:
My thought was that maybe I should just make it consider an empty string without translation mark to be a valid t_string. Does that make sense?